src/app/create/basicauction/page.tsx

```diff
@@ -0,0 +1,207 @@
+
+  const onSubmit: SubmitHandler<AuctionInputs> = data => console.log(data);
+
+  return (
+    <form className={S.auctionForm} onSubmit={handleSubmit(onSubmit)}>
+      <label htmlFor="name" className={S.auctionLabel}>
+        <div className={S.title}>상품명</div>
+        <input
+          className={S.auctionInput}
+          id="name"
+          type="text"
+          placeholder="상품명"
+          {...register('nameRequired', { required: '상품명을 입력해 주세요.' })}
+        />
+        {errors.nameRequired && (
+          <span className={S.error}>
+            <ErrorIcon />
+            {errors.nameRequired.message}
+          </span>
+        )}
+      </label>
+      <label htmlFor="startPrice" className={S.auctionLabel}>
+        <div className={S.title}>시작가</div>
+        <input
+          className={`${S.auctionInput} ${S.priceInput}`}
+          id="startPrice"
+          type="number"
+          placeholder="원"
+          {...register('startPriceRequired', {
+            required: '시작가를 입력해 주세요.',
+            pattern: {
+              value: /^(0|[1-9]\d*)$/,
+              message: '올바른 금액이 아닙니다.',
+            },
+          })}
+        />
+        {errors.startPriceRequired && (
+          <span className={S.error}>
+            <ErrorIcon />
+            {errors.startPriceRequired.message}
+          </span>
+        )}
+      </label>
+      <div className={S.auctionLabel}>
+        <div className={S.title}>사진</div>
+        <div className={S.count}>{fields.length}/10</div>
+        <div className={S.imageBoard}>
+          <ul className={S.imageList}>
+            {fields.map(({ imageRequiredId, file }, index) => (
+              <li key={imageRequiredId} className={S.image}>
+                <img src={URL.createObjectURL(file)} alt={URL.createObjectURL(file)} />
+                <button type="button" className={S.deleteButton} onClick={() => remove(index)}>
+                  <DeleteIcon />
+                </button>
+              </li>
+            ))}
+          </ul>
+          <div className={S.imageUpload}>
+            <label htmlFor="image" className={S.imageInfo}>
+              이미지 등록하기
+              <input
+                className={S.imageInput}
+                ref={inputFile}
+                id="image"
+                type="file"
+                accept="image/*"
+                multiple
+                onChange={addImage}
+              />
+            </label>
+          </div>
+        </div>
+        {errors.imageRequired && (
+          <span className={S.error}>
+            <ErrorIcon />
+            {errors.imageRequired.root?.message}
+          </span>
+        )}
+      </div>
+      <label htmlFor="info" className={S.auctionLabel}>
+        <span className={S.title}>상품 정보</span>
+        <div className={S.count}>
+          {watch('infoRequired') ? watch('infoRequired').length : 0}/500
+        </div>
+        <textarea
+          id="info"
+          className={S.info}
+          maxLength={500}
```

Comment:
```suggestion
          maxLength={499}
```
499로 해야지 딱 500자에서 먹혀들어가네요! 500으로 했을 때는 501에서 안쳐지는 현상이 있습니다

Reply:
이 부분 새로 이슈파서 수정하겠습니다~